generateHash.js

```diff
@@ -0,0 +1,7 @@
+import bcrypt from "bcrypt";
+
+const password = "mcsp2309";
```

Comment:
recommend the seed password be passed as .env variable to improve codebase security